doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -213,6 +253,12 @@ of types inside the syntactic class :token:`term`.
 Qualified identifiers and simple identifiers
 --------------------------------------------
 
+.. insertgram qualid qualid
+
+.. productionlist:: coq
+   qualid : `qualid` `field`
+          : `ident`
```

Comment:
The definition of `field` is only used here if I'm not mistaken. If that's the case, it would make sense to move it here.

Reply:
I'll move it.  It's also used for this (not in the gallina chapter):

```
dirpath: [
| ident
| dirpath field
]
```

Reply:
Hum, that's surprising that another nonterminal `dirpath` is defined that looks completely equivalent to `qualid`...

Reply:
Not really.  The actions are different.